src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -233,6 +235,14 @@
             minValue = 1)
     private int vcfInitializerThreads = 1;
 
+    @Advanced
+    @Argument(fullName = MAX_NUM_INTERVALS_TO_IMPORT_IN_PARALLEL,
+            shortName = MAX_NUM_INTERVALS_TO_IMPORT_IN_PARALLEL,
+            doc = "Max number of intervals to import in parallel; higher values may improve performance",
```

Comment:
maybe edit to -> "higher values may be faster but require more memory"?

Reply:
Done